### DISCUSSION TITLE
I know this setting was there before.

### DISCUSSION THREAD

**Discussion by ACR-Jeff** [jellyfin/jellyfin-androidtv#4078] [General]
I know it was there in previous versions, I can't seem to find it anywhere in Jellyfin-AndroidTV, Let alone in the server, Disable Removing Recently Added when marked watched? Any help would be appreciated, Thanks

**Comment by nielsvanvelzen** [comment]
In the web client go to your "home" settings and uncheck "Hide watched content from 'Recently Added Media'". (very first checkbox on that page). We don't have that preference in the app but we do implement its behavior.

**Comment by ACR-Jeff** [comment]
Thank you, Now I feel like a noob, I looked everywhere and it was right in my face LOL
